empiric-package/empiric/core/config.py

```diff
@@ -0,0 +1,93 @@
+import os
+
+from empiric.core.types import INTEGRATION, MAINNET, STAGING, TESTNET, Network
+from starknet_py.net.models import StarknetChainId
+from typing_extensions import Protocol
+
+
+# TODO (rlkelly): We need a consistent way to implement interfaces.  This seems to be the preferred method
+class IConfig(Protocol):
+    ADMIN_ADDRESS: int
+    DEFAULT_AGGREGATION_MODE: int
```

Comment:
This should be handled separately. We should have a `AGGREGATION_MODES` enum-like object

Reply:
Added one. Do you think the config should not set a default aggregation mode?

Reply:
Not sure that the aggregation mode belongs in the config, though perhaps the default does. Either way it's fine there for now